crates/sui-benchmark/tests/simtest.rs

```diff
@@ -222,6 +222,32 @@ mod test {
         test_simulated_load(TestInitData::new(&test_cluster).await, 120).await;
     }
 
+    #[ignore]
```

Comment:
simtest is disabled for now. It allows finding parts of the system that still rely on past checkpoints/transactions to be present. 
For example, it currently fails with:
```
Can not find effect for transaction {:?}, however transaction that depend on it was already executed
```
I'll enable it in a follow-up PR